.idea/vcs.xml

```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
This looks like it snuck past your .gitignore file. You should be able to run `git rm -r --cached .idea` to get rid of it.